CHANGELOG.md

```diff
@@ -4,8 +4,8 @@
 
 ### Adds
 
-TODO: set documentation link once created:
```

Comment:
They haven't been accepted, yet, so I can't add specific links. How about I add something like:
"See the documentation in the Guide section for 'Areas and Widgets' for further details."?

Reply:
Hmm that I wanted to do in the first place but @boutell  told me that we should have the link, and I agree, for the sake of a better user experience (having the link and not having to look for the documentation is quite cool, because we're all lazy, especially developers 😄 )

We could guess what the link will  be from the title of the documentation page, eg:

```
https://v3.docs.apostrophecms.org/guide/caching.html
```

but that's risky if the page title changes before the merge.
I'd  say we wait for the doc to be merged first :)

Reply:
I checked with https://github.com/apostrophecms/a3-docs/blob/PRO-3236-widget-placeholders/docs/guide/* repo and it seems to be fine, if the titles don't change I guess it's ✅